packages/sdk-middleware-http/src/http.js

```diff
@@ -120,7 +120,7 @@ export default function createHttpMiddleware({
     if (credentialsMode) {
       fetchOptions.credentials = credentialsMode
     }
-    if (timeout || abortController) {
+    if (abortController) {
```

Comment:
FYI: if timeout is set, the abortController will be initialized anyway

Reply:
Good catch 👍